Tests/TemplatesTests/StringsTests.swift

```diff
@@ -33,6 +33,19 @@ class StringsTests: XCTestCase {
     ]
   }
 
+  let variationsObjC: VariationGenerator = { name, context in
+    guard name == "localizable" else { return [(context: context, suffix: "")] }
+
+    return [
+      (context: context,
+       suffix: ""),
+      (context: try StencilContext.enrich(context: context, parameters: ["enumName=XCTLoc"]),
```

Comment:
`enumName` isn't used as far as I can see in the template?

Reply:
yeah, I figured that out too.  good catch.